gptscript/gptscript.py

```diff
@@ -270,6 +270,30 @@ async def add_dataset_element(self, workspace_id: str, datasetID: str, elementNa
         )
         return DatasetElementMeta.model_validate_json(res)
 
+    async def add_dataset_elements(self, workspace_id: str, datasetID: str, elements: List[DatasetElement]) -> str:
```

Comment:
`workspace_id` should be optional
```suggestion
    async def add_dataset_elements(self, datasetID: str, elements: List[DatasetElement], workspace_id: str = "") -> str:
```